bioinflib/bio_files_processor.py

```diff
@@ -0,0 +1,55 @@
+def convert_multiline_fasta_to_oneline(input_fasta: str, output_fasta: str = None) -> None:
```

Comment:
можно было бы добавить для output_fasta аннотацию Optional

```suggestion
from typing import Optional

def convert_multiline_fasta_to_oneline(input_fasta: str, output_fasta: Optional[str] = None) -> None:
```